configs/milvus.yaml

```diff
@@ -435,6 +435,22 @@ dataCoord:
     maxParallelTaskNum: 10 # max parallel compaction task number
     indexBasedCompaction: true
 
+    major:
```

Comment:
About the terminology here：we have major and L2:

I think L2 request user to understand the concept of segment type and relationships between different compaction types and segment types. Here compaction is actually L1 + L2 -> L2,  and we will add L2 single compaction, L2 merge compaction in the future. So I think L2 may cause more confusion. 

For major compaction, we will split one major compaction into multi sub compaction tasks to handle large dataset and handoff. It is a little weird to call these subTask as `major`. We developers should notice that.

I prefer `major` than `L2` for easier user understanding. And another proposal is `clusteringCompaction`.
`clustering` is more accurate and echos the concept of `clusteringKey`

Final compaction type will be like:
  Level0DeleteCompaction
  SingleCompaction:  add support for L2
  MinorCompaction: add support for L2
  ClusteringCompaction(MajorCompaction)： L1+L2 -> L2